website/content/docs/nia/cli/index.mdx

```diff
@@ -61,7 +61,7 @@ list of available completions. To show available flag completes, type `-<tab>`.
 
 Autocompletion will query the running CTS server to return helpful argument suggestions. For example, for the `task disable` command, autocompletion will return the names of all enabled tasks that can be disabled.
 
-When autocompletion makes the query to the running CTS server, it will also use any `CTS_*`` environment variables (for example `CTS_ADDRESS`) set on the CTS server.
+When autocompletion makes the query to the running CTS server, it will also use any `CTS_*` environment variables (for example `CTS_ADDRESS`) set on the CTS server.
```

Comment:
I fixed this in the 0.6.0 docs branch already, just so you're aware

Reply:
Oh yea, I saw your update. I'll leave it and see if GH will resolve it well. Thanks for the heads up

Reply:
Just dropped the commit for this